fitz/fitz.i

```diff
@@ -14412,6 +14419,63 @@ struct Tools
         }
 
 
+        FITZEXCEPTION(set_word_delimiters, !result)
+        %pythonprepend set_word_delimiters %{
+        """Set characters to be word delimiters."""
+        if delims == None:
+            delims = []
+        if not hasattr(delims, "__getitem__") or len(delims) > 64:
+            raise ValueError("bad delimiter value(s)")
```

Comment:
Seems a bit of a shame to have a fixed-size buffer for this.

Reply:
The idea is to allow 'string.punctuation` as delimiters (32 items) plus any equivalent of those crude Chinese characters like "。" (Unicode 0x3002). Nobody will ever be able to keep track of more than this number. We must not forget that none of the delimiters will ever be part of any extracted word ...

Reply:
In rebased, i think it would actually be simpler to implement and document if we used an unlimited Python list.